webui/react/src/hooks/useSettings.ts

```diff
@@ -142,17 +142,19 @@ const useSettings = <T>(config: SettingsConfig<T>): UseSettingsReturn<T> => {
     clearQuerySettings,
   } = useContext(UserSettings);
   const initialLoading = useObservable(isLoadingOb);
-  const [derivedOb] = useState(stateOb.select((s) => s.get(config.storagePath)));
-  const state = useObservable(derivedOb);
+  const settingsByKey = stateOb.select((s) => s.get(config.storagePath));
```

Comment:
This creates a new observable every time the render function is called which orphans any existing subscriptions.